microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -51,14 +143,16 @@ Key compatibility limitations include the operating system and rendering in fram
 <!-- ------------------------------ -->
 #### Operating systems
 
-Windows 7 and Windows 8 can only do windowed hosting, not visual hosting.
+All hosting modes are supported wherever WebView2 is supported ie. Windows 10 and up and certain Windows Server versions. Windows 7, 8 and 8.1 are no longer supported. For those still on those platforms Windows 7 and Windows 8 can only do Windowed hosting, not Visual hosting.
 
 See [Windows 7 and 8](../index.md#windows-7-and-8) in _Introduction to Microsoft Edge WebView2_.
 
 
 <!-- ------------------------------ -->
 #### Rendering WebView2 in framework and non-framework apps
 
+<!-- todo: are these tab lists introduced ok?  are they straight copy of "Overview of APIs"?  link to there instead? -->
```

Comment:
are these tab lists introduced ok?  
are they a straight copy of "Overview of APIs"?   link to there instead?

rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/windowed-vs-visual-hosting?branch=pr-en-us-3271&tabs=dotnetcsharp#rendering-webview2-in-framework-and-non-framework-apps

Reply:
Outdated, entered fresh Comment